src/Symfony/Bridge/Twig/Tests/Mime/NotificationEmailTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Symfony\Bridge\Twig\Tests\Mime;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Twig\Mime\NotificationEmail;
+
+class NotificationEmailTest extends TestCase
+{
+    public function test()
+    {
+        $email = (new NotificationEmail())
+            ->markdown('Foo')
+            ->exception(new \Exception())
+            ->importance(NotificationEmail::IMPORTANCE_HIGH)
+            ->action('Bar', 'http://example.com/')
+            ->context(['a' => 'b'])
+        ;
+
+        $this->assertEquals([
+            'importance' => NotificationEmail::IMPORTANCE_HIGH,
+            'content' => 'Foo',
+            'exception' => true,
+            'action_text' => 'Bar',
+            'action_url' => 'http://example.com/',
+            'markdown' => true,
+            'raw' => false,
+            'a' => 'b',
+        ], $email->getContext());
+    }
+
+    public function testSerialize()
+    {
+        $email = unserialize(serialize((new NotificationEmail())
+            ->markdown('Foo', true)
```

Comment:
You probably mean `->content()` here? then, the expected raw is `true`.

Reply:
fixed